tests/layer_tests/tensorflow_lite_tests/test_tfl_TransposeConv.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+
+from common.tflite_layer_test_class import TFLiteLayerTest
+
+np.random.seed(42)
+
+test_params = [
+    {'shape': [1, 3, 4, 1], 'ksize': [1, 1, 1, 1], 'output_shape': [1, 3, 4, 1], 'strides': [1, 1, 1, 1],
+     'padding': 'SAME', 'data_format': 'NHWC', 'dilations': [1, 1, 1, 1]},
+    {'shape': [1, 4, 4, 1], 'ksize': [1, 1, 1, 1], 'output_shape': [1, 4, 4, 1], 'strides': [1, 1], 'padding': 'SAME',
+     'data_format': 'NHWC', 'dilations': [1, 2, 2, 1]},
```

Comment:
Dilations are absent in the tfl attributes of this op -- I suppose they are all considered = 1s

![image](https://github.com/openvinotoolkit/openvino/assets/55839243/40eaf970-5e16-434c-b2f1-1764fe9fa2a6)